### ISSUE TITLE
"Cache" parameters and log TOC

### ISSUE THREAD

**Issue by fredg02** [bitcraze/crazyflie-clients-python#250]
It would be convenient to see the list of available parameters and logging variables, even after the client has disconnected from the Crazyflie. Right now the tabs are cleared immediately on disconnect.

On these tabs, it should be indicated that the values are old and/or that the Crazyflie is disconnected.

An additional feature would be the option to load/import any cached TOC *.json file from disc.

**Comment by evoggy** [comment]
Yeah, this is definitely possible. It would also be possible to keep the values that were last logged. I think the main issue is how it should look. Maybe graying out all the fields but keeping the data and TOC?

**Comment by fredg02** [comment]
Yes, graying out sounds good!
